flytekit/core/data_persistence.py

```diff
@@ -258,7 +304,10 @@ def is_remote(path: Union[str, os.PathLike]) -> bool:
         """
         Deprecated. Lets find a replacement
         """
-        return not (path.startswith("/") or path.startswith("file://"))  # type: ignore
+        protocol, _ = split_protocol(path)
```

Comment:
should we replace here too - https://github.com/flyteorg/flytekit/blob/master/flytekit/extras/persistence/s3_awscli.py#L85?

Reply:
i guess not needed

Reply:
yeah that should always be a string, although i guess we could move to using split_protocol there